smac/scenario/scenario.py

```diff
@@ -64,13 +66,48 @@ def __init__(self, scenario, cmd_args=None):
             raise ValueError('Could not parse the following arguments: %s' %
                              str(list(scenario.keys())))
 
+        for group, potential_members in self._groups.items():
```

Comment:
If an argument is also set as required this check is not influenced and the code might crash before?
This should be clarified in the docstring: if an argument is part of the required group one should not mark it as required.